lib/__test__/rule-list.test.js

```diff
@@ -380,6 +386,31 @@ describe('rules', () => {
         ['x^-a / x^-b', 'x^(-a - -b)'],
     ])
 
+    suite('multiplying coefficients', rules.MULTIPLY_COEFFICIENTS, [
+        ['x^2 * y^2', '(1 * 1) (x^2 * y^2)'],
```

Comment:
I'm okay with deferring the work to get this behavior until we move this code over to mathsteps.  Can you add a TODO so that we can remind ourselves of this difference in behavior?